db/migrate/20191204094848_add_index_on_submissions_created_at.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexOnSubmissionsCreatedAt < ActiveRecord::Migration[5.2]
+  def change
+    add_index :submissions,
```

Comment:
Is it worth adding the index concurrently, particularly since we're still inside a submission window - e.g. https://semaphoreci.com/blog/2017/06/21/faster-rails-indexing-large-database-tables.html ?